scripts/src/java/org/oppia/android/scripts/common/ScriptExemptions.kt

```diff
@@ -0,0 +1,632 @@
+package org.oppia.android.scripts.common
+
+/**
+ * List of files which should be exempted for the test file check.
+ *
+ * NOTE TO DEVELOPERS: New files should rarely be added here. In general, all new new files should
+ * have tests. If you think there's a valid exception, please specifically call this out in your PR
+ * description.
+ */
+object ScriptExemptions {
```

Comment:
How hard would it be to move this to be a textproto, instead? Keeping all script configurations within textproto files seems kind of nice.

If we went that route, I suggest moving the 'NOTE TO DEVELOPERS' bit as output for the script itself. E.g. something like:

"File X does not have a corresponding test file. If this is correct, please update scripts/assets/script_exemptions.textproto. Note that, in general, all new files should have tests. If you choose to add an exemption, please specifically call this out in your PR description."

WDYT?

Reply:
Sounds good to me. Revamped to the textproto approach. PTAL.